compiler/src/dmd/chkformat.d

```diff
@@ -811,7 +828,11 @@ enum Format
     jn,         // pointer to intmax_t
     zn,         // pointer to size_t
     tn,         // pointer to ptrdiff_t
-    GNU_m,      // GNU ext. : string corresponding to the error code in errno (printf) / length modifier (scanf)
+    GNU_m,      // GNU ext. : string corresponding to the error code in errno (printf)
+    GNU_mc,     // GNU ext. : dynamically allocated char (scanf)
+    GNU_mlc,    // GNU ext. : dynamically allocated wchar_t (scanf)
+    GNU_ms,     // GNU ext. : dynamically allocated char string  (scanf)
+    GNU_mls,    // GNU ext. : dynamically allocated wchar_t string (scanf)
```

Comment:
As you said in the bug report, it's a POSIX extension, so why did you name it as a GNU extension ?
Also, I think it would a good idea to have two separate format `GNU_m` and `POSIX_m` (both `%m` in `GNU_m` is misleading). (or a `GNU_POSIX_m` ?)